static/app/components/replays/virtualizedGrid/useDetailsSplit.tsx

```diff
@@ -1,5 +1,5 @@
 import type {RefObject} from 'react';
```

Comment:
@ryan953 would you mind linking me to where this component is in the UI? I have verified that this works in profiling, but I would like to check for replay as well

Reply:
oh, sorry for the huge delay!

this is imported by the replay network details thing. So if you're looking at a replay, click into the Network tab, then click on a specific network row.... you'll get Network Details coming up from the bottom. It's resizable.

<img width="578" alt="SCR-20250411-jfsd" src="https://github.com/user-attachments/assets/a4be0927-0bab-40b3-b808-5ea11f857e04" />

Reply:
Thanks @ryan953, not a problem about the delay. I have more stuff that I'll need to fix, I just opened a bunch of stacked PRs to get this in, and I'll make sure to test everything where this is used before merging this in